src/Manifest/FileBucketManifestReader.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+declare(strict_types=1);
+
+namespace SprykerSdk\Integrator\Manifest;
+
+use SprykerSdk\Integrator\FileStorage\BucketFileStorageInterface;
+
+class FileBucketManifestReader implements FileBucketManifestReaderInterface
+{
+    /**
+     * @var string
+     */
+    protected const INSTALLER_MANIFEST_JSON = 'installer-manifest.json';
+
+    protected BucketFileStorageInterface $bucketFileStorage;
+
+    /**
+     * @param \SprykerSdk\Integrator\FileStorage\BucketFileStorageInterface $bucketFileStorage
+     */
+    public function __construct(BucketFileStorageInterface $bucketFileStorage)
+    {
+        $this->bucketFileStorage = $bucketFileStorage;
+    }
+
+    /**
+     * @param int $releaseGroupId
+     *
+     * @return array<string, array<string, array<string>>>
+     */
+    public function readManifests(int $releaseGroupId): array
+    {
+        $fileContent = $this->bucketFileStorage->getFile($this->getObjectKey($releaseGroupId));
```

Comment:
AWS SDK has method `doesObjectExist` just if you need to check if the object exists in bucket.

Reply:
++
Thanks 
Added uses for the method